src/game/main.cpp

```diff
@@ -150,15 +154,18 @@ void Check_Windowed(int argc, char *argv[])
         if (strcasecmp(argv[i], "-xpos") == 0) {
             ++i;
             g_xPos = atoi(argv[i]);
-            g_xPos = std::clamp(g_xPos, 0, (int)(Res.right - 800)); // Prevent negative values
         }
 
         if (strcasecmp(argv[i], "-ypos") == 0) {
             ++i;
             g_yPos = atoi(argv[i]);
-            g_yPos = std::clamp(g_yPos, 0, (int)(Res.bottom - 600)); // Prevent negative values
```

Comment:
Did you test this change on windows? I suspect the window positioning now behaves differently than it did before. Before it would clamp only when -xpos or -ypos command line arguments were set. Now it always does it. This looks like it goes beyond the intent of this change, as you change Windows Window behavior and not just add SDL2 Window.